service/src/test/java/greencity/ModelUtils.java

```diff
@@ -4502,6 +4502,46 @@ public static TableColumnWidthForEmployee getTestTableColumnWidth() {
             .build();
     }
 
+    public static TableColumnWidthForEmployee getTestTableColumnWidthForEmployeeWithDefaultValues() {
+        return TableColumnWidthForEmployee.builder()
+            .employee(getEmployee())
+            .address(120)
```

Comment:
do we need to set the width for every column? It is the default value
You can just call constrictor `TableColumnWidthForEmployee(getEmployee())`